typescript/widgets/src/chains/ChainSearchMenu.tsx

```diff
@@ -303,19 +313,22 @@ function useCustomizedListItems(customListItemField) {
       ...(customListItemField ? [customListItemField.header] : []),
       ...Object.values(ChainSortByOption),
     ],
-    [customListItemField],
+    [defaultSortField],
```

Comment:
This looks incorrect. `customListItemField` is used in the memo fn but `customListItemField` isn't. I would have expected the linter to catch this

Reply:
Fixed dependecy